### PR TITLE
First version of Nucleus Tree Creator

### PR DESCRIPTION
**Pull request by fbellini**

### CODE REVIEW

File: PWGLF/TableProducer/LFTreeCreatorNuclei.cxx
Comment (njacazio) [inline PWGLF/TableProducer/LFTreeCreatorNuclei.cxx:142]: this you can probably set it via: `collision.bc().runNumber()`

File: PWGLF/TableProducer/LFTreeCreatorNuclei.cxx
Comment (njacazio) [inline PWGLF/TableProducer/LFTreeCreatorNuclei.cxx:131]: Here you could subscribe to BCs  `aod::BC const&` to have the info about the run number later

File: PWGLF/TableProducer/LFTreeCreatorNuclei.cxx
Comment (njacazio) [inline PWGLF/TableProducer/LFTreeCreatorNuclei.cxx:141]: Since this is supposed to be working on Run3 data we can start adding the ev. sel. statement e.g. `collision.sel8()`

**Review by njacazio — COMMENTED** [review]
@fbellini great, thanks a lot! I'm still wondering if TableProducer is the best place for this task as it just provides filtering for the data and it is not used by other tasks.
What do you think, shall we move it to tasks?

File: PWGLF/TableProducer/LFTreeCreatorNuclei.cxx
Comment (fbellini) [reply PWGLF/TableProducer/LFTreeCreatorNuclei.cxx]: Hi @njacazio, I have set runnumber as suggested. 
On the folder to store the tree creator, I have chosen TableProducer as it is done similarly in HF.
We can move it to tasks, if you believe it is a better place.

File: PWGLF/TableProducer/LFTreeCreatorNuclei.cxx
Comment (fbellini) [reply PWGLF/TableProducer/LFTreeCreatorNuclei.cxx]: See above.

File: PWGLF/TableProducer/LFTreeCreatorNuclei.cxx
Comment (fbellini) [reply PWGLF/TableProducer/LFTreeCreatorNuclei.cxx]: Done.

**Review by njacazio — DISMISSED** [review]

**Comment by njacazio** [comment]
We need still to fix the formatting before merging

File: PWGLF/TableProducer/LFTreeCreatorNuclei.cxx
Comment (iarsene) [reply PWGLF/TableProducer/LFTreeCreatorNuclei.cxx]: Hi @njacazio and @fbellini,
I would say that this task can stay in TableProducer since it indeed produces skims. However, for these skims to be used properly in a consumer task, one has to move the data model definition to a separate header, which can be visible to more than just this task. I guess this can be the topic of a future PR. 
I also see that this PR is modifying the aodMerger.cxx. Is this really intended ? Then maybe @jgrosseo should have a look.

Comment (jgrosseo) [reply PWGLF/TableProducer/LFTreeCreatorNuclei.cxx]: Thanks for flagging. This must be the result of an invalid merge. @fbellini please check before making the PR if you touch unexpected files. Github nicely shows the changes and one can scroll through...

Comment (jgrosseo) [reply PWGLF/TableProducer/LFTreeCreatorNuclei.cxx]: I think it sneaked in later when you pushed 
image

Comment (fbellini) [reply PWGLF/TableProducer/LFTreeCreatorNuclei.cxx]: @jgrosseo, @iarsene hi! Thank you for spotting this mistake and sorry for messing up with my previous push. It was due to a misconfiguration of my fork. Hopefully my settings are ok now and it should not happen again. 
To answer @iarsene, yes, I can provide the separate header in a future PR. Thanks for the suggestion.

**Review by njacazio — APPROVED** [review]
Perfect, this PR is now, clean! Thanks a lot!

**Review by njacazio — APPROVED** [review]

**Pull request merged**
